app/workers/mentions/create_all_worker.rb

```diff
@@ -4,6 +4,7 @@ class CreateAllWorker
     sidekiq_options queue: :default, retry: 10
 
     def perform(notifiable_id, notifiable_type)
+      # This worker is currently only used to create mentions on comments.
```

Comment:
I think this could also make good class-level documentation which again would get picked up by YARD. But maybe I'm actually the only person here who locally generates docs.